core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -282,6 +282,11 @@ private[spark] class SecurityManager(sparkConf: SparkConf)
       }: TrustManager
     })
 
+    if (!fileServerSSLOptions.protocol.isDefined) {
```

Comment:
Oh, I liked `require` here, was just suggesting using `isDefined`.
Also the following line now doesn't just call `.get` which was also the right change.